### PR TITLE
add links to contributors guide

### PR DESCRIPTION
**Pull request by notsureifkevin**
### What does this PR do?

- updates the maintainers documentation to reference the contributors guide repo
- updates the submitting-pull-requests documentation to ref. the contributors guide repo

### Motivation

the contributors guide repo being made public!

### More

- [ ] Added/updated tests
- [x] Added/updated documentation

### CODE REVIEW

**Comment by jbdoumenjou** [comment]
In my opinion, it is not necessary to remove this part of the documentation from the Traefik project.
Adding a link (for more details/ if you want more ...) to the experimental repository contributor guide is sufficient.

File: docs/content/contributing/maintainers.md [hunk 1]
```diff
@@ -18,68 +18,13 @@
...
-### [Myrmica Aloba](https://github.com/traefik/aloba)
-
-Manage GitHub labels.
+## PR review process:
 
-* Add labels on new PR [GitHub WebHook]
-* Add milestone to a new PR based on a branch version (1.4, 1.3, ...) [GitHub WebHook]
-* Add and remove `contributor/waiting-for-corrections` label when a review request changes [GitHub WebHook]
-* Weekly report of PR status on Slack (CaptainPR) [cron]
+PRs are reviewed during the same daily meeting as our Triage, and the process for reviewing PRs may be found under [review guidelines](https://github.com/traefik/contributors-guide/blob/master/review_guidelines.md) in our contributors guide repository.
```
Comment (rtribotte) [inline docs/content/contributing/maintainers.md:27]: I think it's better to just give the link to the review process, the PR triage during the daily is already mentioned above and the term `review` here is confusing to talk about triage/status update.

```suggestion
The process for reviewing PRs may be found under [review guidelines](https://github.com/traefik/contributors-guide/blob/master/review_guidelines.md) in our contributors guide repository.
```

**Comment by notsureifkevin** [comment]
> 
> 
> In my opinion, it is not necessary to remove this part of the documentation from the Traefik project.
> Adding a link (for more details/ if you want more ...) to the experimental repository contributor guide is sufficient.

The content that I removed was duplicated in the new contributors guide repo, and for the benefit of the reader, it seemed logical to present the information in one place rather than two. Does it make more sense to have the information in both places?

**Comment by jbdoumenjou** [comment]
Yes, it seemed coherent to me because the contributor can have a quick view in the Traefik project directly and go to see the details and/or participate in proposing improvements on the contributor guide.
Having said that, in both cases it suits me. :smile:

**Review by jbdoumenjou — APPROVED** [review]
LGTM :+1:

**Review by rtribotte — APPROVED** [review]
LGTM 👌

**Review by SantoDE — APPROVED** [review]
LGTM 👼

**Pull request merged**